rust/processor/src/schema.rs

```diff
@@ -1197,6 +1198,10 @@ diesel::table! {
 }
 
 diesel::joinable!(block_metadata_transactions -> transactions (version));
+diesel::joinable!(move_modules -> transactions (transaction_version));
+diesel::joinable!(move_resources -> transactions (transaction_version));
+diesel::joinable!(table_items -> transactions (transaction_version));
+diesel::joinable!(write_set_changes -> transactions (transaction_version));
```

Comment:
can you remove these manually for now? We have a PR coming that removes it once for all.

Reply:
done.